WebHostLib/templates/hostRoom.html

```diff
@@ -3,6 +3,16 @@
 {% block head %}
     <title>Multiworld {{ room.id|suuid }}</title>
     {% if should_refresh %}<meta http-equiv="refresh" content="2">{% endif %}
+    <meta name="og:site_name" content="Archipelago">
+    <meta property="og:title" content="Multiworld {{ room.id|suuid }}">
+    <meta property="og:type" content="website" />
+    {% if room.seed.slots|length < 2 %}
+    <meta property="og:description" content="{{ room.seed.slots|length }} Player World
+    {% if room.last_port != -1 %}running on {{ config['HOST_ADDRESS'] }} with port {{ room.last_port }}{% endif %}">
+    {% else %}
+    <meta property="og:description" content="{{ room.seed.slots|length }} Players Multiworld
+    {% if room.last_port != -1 %}running on {{ config['HOST_ADDRESS'] }} with port {{ room.last_port }}{% endif %}">
+    {% endif %}
```

Comment:
```suggestion
    {% if room.seed.slots|length < 2 %}
    <meta property="og:description" content="{{ room.seed.slots|length }} Player World
    {% if room.last_port != -1 %}running on {{ config['HOST_ADDRESS'] }} with port {{ room.last_port }}{% endif %}">
    {% else %}
    <meta property="og:description" content="{{ room.seed.slots|length }}-Player Multiworld
    {% if room.last_port != -1 %}running on {{ config['HOST_ADDRESS'] }} with port {{ room.last_port }}{% endif %}">
    {% endif %}
```

Small nitpick, but with the most recent format of the embed, for rooms of mutliple people, "Players" does not work in this situation. Player is describing the room, not acting as an object, my change modifies the output so that it reads as
"12-Player Multiworld" rather than "12 Players Multiworld".

Reply:
I'm thoroughly confused by this

Reply:
Perhaps I phrased it poorly
In the screenshot, there are 2 formats that you have for the embed. For the first screenshot, the description has a line for the player count and a line for "Multiworld running on [host] with port [port]". The second screenshot, which is the one that the code currently represents, moves "Multiworld" up onto the line with the palyer count.

Because of this, the description looks like it says that the link is to a 12 Player Multiworld, not that it is 12 Players, Multiworld. What I am attempting to suggest is that currently, "12 Players Multiworld" seems incorrect to me. I believe that it should be "12-Player Multiworld" or "12 Player Multiworld" since the formatting makes it look like it focuses on the multiworld, not the player count. In other words, the player count is a description of the multiworld.

I hope my clarification helps